generated/nidigital/session.py

```diff
@@ -2830,7 +2830,12 @@ def _fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, tim
     def fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, timeout):
         '''fetch_capture_waveform
 
-        TBD
+        Returns a list of named tuples (Waveform) that <FILL IN THE BLANK HERE>
```

Comment:
We should add an issue about `<FILL IN THE BLANK HERE>` unless we have another way to track it in place.